docs/03_linki.md

```diff
@@ -0,0 +1,44 @@
+# Społeczność, fora i czaty
+
+## Polska
+- (czat) przestrzeń MeshtasticPL na komunikatorze Matrix https://matrix.to/#/#meshtasticpl:matrix.org
+- Mapa (i serwer MQTT) https://loranet.pl
+- https://lora.waw.pl/
+  - forum https://lora.waw.pl/forum/
+- https://przejdznamediumfast.pl/
+- Forum https://meshtastic.pop.pl/
```

Comment:
Strona jednego z użytkowników, za wiele tam nie ma poza zrobieniem nodów na podstawie https://github.com/NanoVHF/Meshtastic-DIY. Można ewentualnie to dodać do Konstrukcje stacji DIY.

Reply:
Wiem jak jest, tym niemniej jest to forum jak każde inne i może się rozwinie albo nie. Mogę przenieść na koniec listy :P